typescript/test/testdata/echo_client_baseline

```diff
@@ -0,0 +1,234 @@
+// Copyright 2019 Google LLC
```

Comment:
This isn't blocking feedback.  For these baselines - should we consider using something similar to the snapshotting model @bcoe is using in other cmdline apps?

Reply:
here's an example of using snapshots to test a CLI:

https://github.com/bcoe/c8/blob/master/test/integration.js#L18

It can be a really nice pattern if you have large payloads you're asserting against, updating tests becomes more visual. Where, if you make a change, and tests start failing, you eyeball the snapshot to make sure that the change in formatting was expected, if it was you simply run `test:snap`, to recapture snapshots.

What's nice is you automate the process of maintaining large fixtures.

Reply:
I suggest we merge this one as is just because I want to have *some* baseline test ASAP, and then change it to a better engineered version. For now, I just need to be confident when making change, so I asked @xiaozhenliu-gg5 to add the very basic baseline test.